src/pages/CollectionPage/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"
+import { Container, Row, Col, Card } from 'react-bootstrap';
+import { Collection } from "@imtbl/core-sdk";
+import HeaderBar from "../../components/HeaderBar/HeaderBar"
+import { getCollections } from "../../api/collections-api";
+import { Link } from "react-router-dom";
+import placeholderImg from '../../asset/placehoderImg.jpg';
+
+const CollectionsPage = () => {
+    const [result, setResult] = useState<Collection[]>([])
```

Comment:
Do the names of these constants accurately represent the data that they store or act on?

One thing to keep in mind is that we want to write code _for humans_. The compiler doesn't really care what this is called.

If you looked at this component without having any context, what would help you immediately understand the intention?

Reply:
Another hint:

If you had to add another API call to this component, would you then call it result2?